examples/all-clusters-app/esp32/README.md

```diff
@@ -53,17 +53,11 @@ step. To install these components manually, follow these steps:
 Currently building in VSCode _and_ deploying from native is not supported, so
 make sure the IDF_PATH has been exported(See the manual setup steps above).
 
--   In the root of the example directory, sync the CHIP tree's submodules and
-    source `idf.sh`. Note: This does not have to be repeated for incremental
-    builds.
-
-          $ source idf.sh
-
 -   Configuration Options
 
         To choose from the different configuration options, run menuconfig
 
-          $ idf make menuconfig
+          $ idf.py menuconfig
```

Comment:
Something above should talk about setting up PATH so this is in it, right?

Reply:
@sweetymhaiske  In all the examples, the `./install.sh` should be followed by `. ./export.sh` command

Reply:
Added.